CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fix some assertions [#129](https://github.com/svenstaro/bvh/pull/129) (thanks @finnbear)
 - Fix traversal in case of single-node BVH [#130](https://github.com/svenstaro/bvh/pull/130) (thanks @finnbear)
 - Fix intersection between rays and AABBs that lack depth. [#131](https://github.com/svenstaro/bvh/pull/131) (thanks @finnbear)
+- Document the limitations of distance traversal best-effort ordering. [#135](https://github.com/svenstaro/bvh/pull/135) (thanks @finnbear)
```

Comment:
Well, it's not really a code change so I don't think we need a CHANGELOG entry for this.

Reply:
Hmm, I don't feel strongly about this, but it does seem worth noting. The function changed from claiming to produce ordered results (while having a bug) to claiming to produce not-necessarily-ordered results (while not having a bug). Even though the source code didn't changed, any user of the function should be put on notice to ensure their use-case doesn't depend on a property that doesn't hold.

Also, mentioning docs in changelogs is [not unprecedented](https://github.com/mrdoob/three.js/releases).

Reply:
Works for me.